analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -86,7 +86,8 @@ private fun SpdxDocument.isProject(): Boolean = projectPackage() != null
  * defined.
  */
 private fun SpdxDocument.projectPackage(): SpdxPackage? =
-    packages.takeIf { it.size > 1 }?.find { it.packageFilename.isEmpty() || it.packageFilename == "." }
+    packages.takeIf { it.size > 1 || (it.size == 1 && externalDocumentRefs.isNotEmpty()) }
+        ?.find { it.packageFilename.isEmpty() || it.packageFilename == "." }
```

Comment:
Remark: I would feel better if there was a test case for this constellation - but I won't insist.